examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -6360,7 +6360,6 @@ endpoint 0 {
     ram      attribute supportsDNSResolve default = false;
     callback attribute generatedCommandList;
     callback attribute acceptedCommandList;
-    callback attribute eventList;
```

Comment:
Fwiw, I am 99% sure the .zap and .matter changes here don't matter: to save space we don't codegen these global attributes in any way and the SDK special-cases them instead of using the ZAP config for them.

So in practice, the only changes I suspect we really need are the change to BUILD.gn, the new PICS, and the changes to the YAML tests.

Reply:
Filed https://github.com/project-chip/zap/issues/1085

Reply:
OK.  It just makes the PR a lot bigger and the key changes less visible, but fine.